### ISSUE TITLE
 Use  TiDB-Ansible Upgrade 

### ISSUE THREAD

**Issue by 799983050** [pingcap/tidb#10613]
![图片](https://user-images.githubusercontent.com/45658355/58445271-a1877a80-812e-11e9-85ef-1dfd85e38b4a.png)

 The current version of tidb is 2.1.1
 Operate according to the document ， Upgraded version to tidb-v2.1.10。
$ wget http://download.pingcap.org/tidb-v2.1.10-linux-amd64.tar.gz
 The upgrade process runs smoothly and without errors 。
but，SELECT tidb_version().
=================================
Release Version: v2.1.6
Git Commit Hash: 66e639e8e127f71662f37287eb984dd84d169cba
Git Branch: HEAD
UTC Build Time: 2019-03-15 09:09:07
GoVersion: go version go1.11.2 linux/amd64
Race Enabled: false
TiKV Min Version: 2.1.0-alpha.1-ff3dd160846b7d1aed9079c389fc188f7f5ea13e
Check Table Before Drop: false
==========================
 What is the reason?

**Comment by liubo0127** [comment]
It seems that the wrong version may be used. Confirm the version with `tidb-ansible/resources/bin/tidb-server -V`. The version of 2.1.10 should be:
```
Release Version: v2.1.10
Git Commit Hash: 329afdadfc0feacbc92696fde5ebee5dac0638ed
Git Branch: HEAD
UTC Build Time: 2019-05-21 03:44:29
GoVersion: go version go1.12 linux/amd64
Race Enabled: false
TiKV Min Version: 2.1.0-alpha.1-ff3dd160846b7d1aed9079c389fc188f7f5ea13e
Check Table Before Drop: false
```

**Comment by 799983050** [comment]
@liubo0127 
$ wget http://download.pingcap.org/tidb-v2.1.10-linux-amd64.tar.gz
We used this link in 2.1.10. Is this wrong? Can you provide download information for the correct version?

**Comment by liubo0127** [comment]
This link is correct, and the tidb version in this package is as follows:
```
Release Version: v2.1.10
Git Commit Hash: 329afdadfc0feacbc92696fde5ebee5dac0638ed
Git Branch: HEAD
UTC Build Time: 2019-05-21 03:44:29
GoVersion: go version go1.12 linux/amd64
Race Enabled: false
TiKV Min Version: 2.1.0-alpha.1-ff3dd160846b7d1aed9079c389fc188f7f5ea13e
Check Table Before Drop: false
```

**Comment by 799983050** [comment]
So why did my upgraded version become 2.1.6?
Do you need any other operation? Or you can give some useful advice.

**Comment by liubo0127** [comment]
Have you checked the version of `tidb-server` in the downloaded package?

**Comment by 799983050** [comment]
![图片](https://user-images.githubusercontent.com/45658355/58447150-1a3e0500-8136-11e9-938b-0c6acb36d1b4.png)
 Why is that?

**Comment by liubo0127** [comment]
Is the operation procedure as follows?
```
1. wget http://download.pingcap.org/tidb-v2.1.10-linux-amd64.tar.gz
2. tar xf tidb-v2.1.10-linux-amd64.tar.gz
3. cp tidb-v2.1.10-linux-amd64/bin/* tidb-ansible/resources/bin/
```

Here is the correct version, you can verify it
![image](https://user-images.githubusercontent.com/28344969/58447386-c54ebe80-8136-11e9-8ffc-1d7e1891fd5c.png)

**Comment by 799983050** [comment]
Yes, the same operation 。

**Comment by 799983050** [comment]
We need to manually copy tidb-server, tikv-server, pd-server commands to the corresponding machine to restart the tidb service？

**Comment by liubo0127** [comment]
The manual update mode has been written, you can see the screenshot of the document you pasted. Of course we still don't recommend manual replacement.
![image](https://user-images.githubusercontent.com/28344969/58448365-46f41b80-813a-11e9-83c9-ac1038d2be76.png)

**Comment by 799983050** [comment]
@liubo0127 what? Look at the first picture of my question. That's how I do it.

**Comment by 799983050** [comment]
What is the reason for this?  I did the right thing.

**Comment by liubo0127** [comment]
According to this screenshot test, what is the version? If the version is `2.1.10` and the `tidb-ansible/resources/bin/tidb-server` version is `2.1.6`, is that an operation error?

![image](https://user-images.githubusercontent.com/28344969/58458433-cfcf7f00-815b-11e9-944b-7dbf9eae8425.png)

**Comment by 799983050** [comment]
That's my question，why？ My upgraded version is V2.1.10 。
 Why did the last select tidb_version() be V2.1.6 .

**Comment by liubo0127** [comment]
> That's my question，why？ My upgraded version is V2.1.10 。
> Why did the last select tidb_version() be V2.1.6 .

Sorry, I didn't see your confirmation before. 
Log in to the tidb machine, `tidb-server -V` to see the version. if it is 2.1.10, maybe the result of `select tidb_version();` is a bug

**Comment by 799983050** [comment]
Solved，tidb-ansible Update to 2.1.10.
